vars/CentOS_7.yml

```diff
@@ -6,3 +6,7 @@ blivet_package_list:
   - libblockdev-lvm
   - libblockdev-mdraid
   - libblockdev-swap
+__storage_blivet_mkfs_option_map:
```

Comment:
Only `BlivetDiskVolume` uses this map. Shouldn't the map also have something to indicate which volume type the arguments apply to?